packages/mssql/src/connection-manager.ts

```diff
@@ -52,10 +52,6 @@ export class MsSqlConnectionManager extends AbstractConnectionManager<
       options: removeUndefined(inlinedOptions),
     };
 
-    if (!tediousConfig.options!.port) {
```

Comment:
Why not add an additional check for usage of `instanceName`?

Reply:
@WikiRik I could do but I don't think it's needed at all, I've checked it without an `instanceName` and `port` and it still defaults to `1433` in tedious